core/src/main/java/org/opensearch/sql/planner/physical/NestedOperator.java

```diff
@@ -128,49 +117,38 @@ public ExprValue next() {
   }
 
   /**
-   * Generate list of non-nested fields that are in inputMap, but not in the member variable
-   * fields list.
+   * Generate list of non-nested fields that are in inputMap, but not in the member variable fields
+   * list.
+   *
    * @param inputMap : Row to parse non-nested fields.
    */
   public void generateNonNestedFieldsMap(ExprValue inputMap) {
     for (Map.Entry<String, ExprValue> inputField : inputMap.tupleValue().entrySet()) {
       boolean foundNestedField =
-          this.fields.stream().anyMatch(
-            field -> field.split("\\.")[0].equalsIgnoreCase(inputField.getKey())
-          );
+          this.fields.stream()
+              .anyMatch(field -> field.split("\\.")[0].equalsIgnoreCase(inputField.getKey()));
 
       if (!foundNestedField) {
         this.nonNestedFields.add(inputField.getKey());
       }
     }
   }
 
-
   /**
-   * Simplifies the structure of row's source Map by flattening it,
-   * making the full path of an object the key
-   * and the Object it refers to the value.
+   * Simplifies the structure of row's source Map by flattening it, making the full path of an
+   * object the key and the Object it refers to the value.
    *
-   * <p>Sample input:
-   * keys = ['comments.likes']
-   * row = comments: {
-   * likes: 2
-   * }
+   * <p>Sample input: keys = ['comments.likes'] row = comments: { likes: 2 }
```

Comment:
Add `pre` or few `br`

Reply:
Thanks! added `pre` in d52b65cd9fbbf4aa5599a663c3872f896f27daca